Automattic/jetpack
# Update/vaultpress backup card purchased state

## Proposed changes:

* Add most recent undoable activity log item to purchased state of VaultPress backup card
* Add undo button
* Add ability to add additional actions to the My Jetpack cards (such as the Undo action) in a dropdown
* Show when activity was last detected

### Other information:


## Jetpack product discussion

P2: p1HpG7-pn9-p2

## Does this pull request change what data or activity we track or use?

Yes, we will be tracking when users click the Undo button and will track the Undo ID within that event

## Testing instructions:

1. Create an Ephemeral site with the Jetpack beta plugin and checkout this branch
2. Connect your site to Jepack, but do not purchase a Backup plan yet
3. Head to `/wp-admin/admin.php?page=my-jetpack` and you should see something like this depending the assets your site has
![image](https://github.com/Automattic/jetpack/assets/65001528/53ff9fb4-a6e7-4632-9268-e6730bd0b222)
4. Go to the Store Admin and assign a VaultPress Backup (daily) deprecated plan to your site. This is to make sure we don't see anything weird if the site doesn't have realtime backup capabilities
5. Go back to My Jetpack and you should see this
![image](https://github.com/Automattic/jetpack/assets/65001528/015bd186-5c73-408a-ae39-9ba1b7de36e9)
6. Make sure your site runs a daily backup and make sure you still don't see anything on the card
7. Next, remove the VaultPress Backup (daily) subscription and purchase a current VaultPress Backup plan
8. Add an asset or two and make sure a realtime backup is created for that event
9. Head over to My Jetpack and you should see the most recent event backed up along with the Undo button
![image](https://github.com/Automattic/jetpack/assets/65001528/45743e7f-1ffe-42f8-a074-102bd89d50b4)
10. Make sure the Manage CTA still works by selecting the dropdown and choosing Manage, and then clicking the CTA
![image](https://github.com/Automattic/jetpack/assets/65001528/474a96f2-b0a6-4074-8d92-fd2ef1b6f80b)
11. Now click the Undo CTA and make sure you are taken to the cloud with the correct realtime backup to restore to
